react/week4/github_users_search/src/UserRepo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import searchContext from "./context";
+import './index.css';
+
+function UserRepo() {
+    const { reposArray, loginName, handleClickRepos, isClickedRepos } = React.useContext(searchContext);
+
+    return (
+        <div>
+            {console.log(isClickedRepos)}
+            {isClickedRepos ? <h3>Cannot fetch more details</h3> :
+                <div>
+                    <h3>Repositories of {loginName}</h3>
+                    {reposArray.map((user, index) =>
+                        <li key={index} className='github_repolist'>
+                            <strong><label>Name of the repository:</label></strong><a href={user} onClick={(event) => handleClickRepos(event)}>{user.name}<br />
+                                <strong><label>Description:</label></strong><span>{user.description}</span><br />
+                                <strong><label>Created date:</label></strong><span>{user.created_at}</span><br />
+                                <strong><label>Open issues:</label></strong><span>{user.open_issues}</span><br />
+                            </a></li>)}
+                </div>}
+        </div>
+    )
+}
+
+export default UserRepo;
+
+// {
```

Comment:
No need to keep commented code around in the commit. :) In general, commented code should be removed (in git).